include/assimp/mesh.h

```diff
@@ -268,6 +271,12 @@ struct aiBone {
     //! The maximum value for this member is #AI_MAX_BONE_WEIGHTS.
     unsigned int mNumWeights;
 
+    // The bone armature node - used for skeleton conversion
+    C_STRUCT aiNode* mArmature;
+
+    // The bone node in the scene - used for skeleton conversion
```

Comment:
This updates core types in assimp and is going to be available to reduce complexity when making skeletal data or mesh skinning.